docs/application/tizen-studio-0/setup/install-sdk.md

```diff
@@ -102,4 +102,4 @@ package-manager-cli install [--accept-license] [--no-java-check] [--proxy <value
 
 ## Related information
 - Dependencies
-  - Tizen Studio 1.0 and Higher
+  - Tizen Studio 1.0 and Higher
```

Comment:
Just for the reference..

tizen-studio-0 folder is not in use. So you don't need to update the files in this folder.

Reply:
noted.